extensions/amp-font/0.1/amp-font.js

```diff
@@ -124,7 +128,7 @@ export class AmpFont extends AMP.BaseElement {
       this.onFontLoadSuccess_();
     }).catch(error => {
       this.onFontLoadError_();
-      throw error;
+      log.warn(TAG_, error, this.element);
```

Comment:
@dvoytenko suggested log.warn - as opposed to console.warn. 

I have added a warn. @cramforce Comment on which one needs removal now

Ideally it would be nice to have log.warn decide which ones should be console.warn'd, rather than using 2 different logging systems